sonar-apple-plugin/src/main/java/fr/insideapp/sonarqube/apple/xcode/runner/XcodeResultReadObjectRunner.java

```diff
@@ -32,6 +32,7 @@ public XcodeResultReadObjectRunner() {
     protected String[] arguments(File resultBundle, Reference reference) {
         return new String[]{
                 "xcresulttool", "get",
+                "--legacy",
```

Comment:
will it still work with Xcode < 16 ?

Reply:
Indeed, I check and it doesn't work.
I checked how other tools managed to handle both.

- fastlane: result of `xcrun xcresulttool version` and expect a minimum version (23021).
https://github.com/fastlane/fastlane/pull/22147/files#diff-3c9438dd39c9680b4944d642b8eac74d953cd446703bb1901842d7fa03f8c248R217

- xcresulttool: result of `xcodebuild -version` and expect a minimum version (16).
https://github.com/gemmakbarlow/xcresulttool/pull/1/files#diff-abe2818e9c29aadf84401cb7ef8286c7cbce9253eba09f6fa52e22ca4e57b17eR82

Which solution seems to be the best for us?

Reply:
`xcrun xcresulttool version`seems the easier to parse :)

Reply:
@gaelfoppolo Hello! 
Do you have any updates on this issue?
I'm looking forward to using a fixed version.

Reply:
I implemented a checker based on what fastlane does. Tested with Xcode 15 and 16, both are now working.